src/lib.rs

```diff
@@ -125,6 +125,19 @@ impl FloatPredicate {
     }
 }
 
+
+/// Defines the optimization level used to compile a `Module`.
+/// 
+/// # Remarks
+/// See also: http://llvm.org/doxygen/CodeGen_8h_source.html
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+pub enum OptimizationLevel {
```

Comment:
I think you can do something like `#[repr(u32)]` so that you don't have to cast the enum as a u32?